.github/workflows/maven.yml

```
@@ -86,10 +86,10 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2
           restore-keys: ${{ runner.os }}-m2
-      - name: Set up JDK 21
+      - name: Set up JDK 25
         uses: actions/setup-java@v2
```

actionlint (1.7.11)

[error] the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)